Unit-Tests/Replication_Tests.m

```diff
@@ -606,23 +606,23 @@ - (void) test10_ReplicationCookie {
                                    NSHTTPCookieOriginURL: remoteDbURL,
                                    NSHTTPCookiePath: remoteDbURL.path,
                                    NSHTTPCookieValue: @"logmein",
-                                   NSHTTPCookieExpires: [NSDate dateWithTimeIntervalSinceNow: 10]
+                                   NSHTTPCookieExpires: [NSDate distantFuture]
```

Comment:
Note: Not related to this fix, but I think this will prevent intermittent failures in this unit test. (I got one of those failures just now.)